### PR TITLE
Me permissions update

### PR DESCRIPTION
**Pull request by ZASMan**
### Master Pivotal ticket
* https://www.pivotaltracker.com/n/projects/2481183/epics/4808935

### CODE REVIEW

File: app/models/permission.rb [hunk 1]
```diff
@@ -48,6 +48,12 @@ class Permission
   field :can_access_age_off_excluded, type: Boolean, default: false
   field :can_send_secure_message, type: Boolean, default: false
   field :can_manage_qles, type: Boolean, default: false
+  field :can_edit_aptc, type: Boolean, default: false
```
Comment (ZASMan) [inline app/models/permission.rb:51]: Ok good, you've added all these as default to false so we should be good ot go

File: features/step_definitions/integration_steps.rb [hunk 2]
```diff
@@ -266,7 +266,7 @@ def non_dc_office_location
                             can_complete_resident_application: true, can_add_sep: true, can_view_username_and_email: true, can_view_application_types: true,
                             view_personal_info_page: true, can_access_outstanding_verification_sub_tab: true, can_access_identity_verification_sub_tab: true,
                             can_access_accept_reject_paper_application_documents: true, can_delete_identity_application_documents: true,
-                            can_access_accept_reject_identity_documents: true)
+                            can_access_accept_reject_identity_documents: true, can_edit_aptc: true)
```
Comment (ZASMan) [inline features/step_definitions/integration_steps.rb:269]: This was just for ME right?

Even though this is a cucumber we should probably use `tap` to add it in conditionally if its configured.

Comment (nks2109) [reply features/step_definitions/integration_steps.rb:269]: No, it is for both ME and DC. Added a new attribute in the permission model, earlier we were using the can_update_ssn attribute for various features to turn permission level on or off now `Edit APTC / CSR` using its own level of permission attribute with can_edit_aptc.

**Review by bill-transue — APPROVED** [review]
Looks good, I like the `role` refactor. Great work @nks2109 and @ZASMan.

**Pull request merged**